matchzoo/datapack.py

```diff
@@ -36,23 +46,24 @@ class DataPack(object):
     DATA_FILENAME = 'data.dill'
 
     def __init__(self,
-                 relation: typing.Union[list, np.ndarray],
-                 content: dict,
-                 context: dict={},
-                 columns: list=None):
+                 relation: pd.DataFrame,
+                 left_data: pd.DataFrame,
+                 right_data: pd.DataFrame,
```

Comment:
I don't like the variable name, i.e. `left_data` and `right_data`. If I were you I would use `left` and `right` directly.

Reply:
It's a little redundant to use `left_data`. `left` is better~